sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -844,6 +844,18 @@ public static void endSession() {
     return getCurrentHub().isCrashedLastRun();
   }
 
+  /**
+   * Report a screen has been fully loaded. That means all data needed by the UI was loaded. If
+   * time-to-full-display tracing {{@link SentryOptions#isEnableTimeToFullDisplayTracing()} } is
+   * disabled this call is ignored.
+   *
+   * <p>This method is safe to be called multiple times. If the time-to-full-display span is already
+   * finished, this call will be ignored.
+   */
+  public static void reportFullDisplayed() {
+    getCurrentHub().reportFullyDrawn();
```

Comment:
`m`: should we align the naming, i.e. rename everything from `fullyDrawn` to `fullDisplayed`?

Reply:
yeah, i renamed the "external" API and forgot to update the hub 😅